lib/HttpClient.js

```diff
@@ -698,6 +731,8 @@ HttpClient.prototype._options = function (method, options) {
         cert: options.cert || self.cert,
         ciphers: options.ciphers || self.ciphers,
         connectTimeout: options.connectTimeout || self.connectTimeout,
+        exponentialBackoff: typeof options.exponentialBackoff !== 'undefined' ?
```

Comment:
`options.exponentialBackoff !== undefined`?

Reply:
By default the value should be true, so the user sets it with `false` which means I can't do `false || true`. There's probably a more elegant way to do this but this is pretty explicit.

Reply:
Ahh okay, my bad. Avoid me